.github/workflows/test-prom-version-upgrade.yaml

```diff
@@ -9,6 +9,11 @@ jobs:
     name: Upgrade Prometheus
     runs-on: ubuntu-latest
     steps:
+    - name: Reclaim disk space
+      run: |
+        docker image prune --force --all
+        sudo rm -rf /usr/share/dotnet
```

Comment:
Are these directories by default in runner?

Reply:
yes

Reply:
I have a plan to strip down all unnecessary content from the ubuntu image but it's for a follow-up.